### PR TITLE
add _edm4hep tutorials

### PR DESCRIPTION
**Pull request by clementhelsens**
Hello @gganis , @vvolkl 

I have created two new files with edm4hep tutorials to replace the fccedm ones.
I have checked that it runs, but more eyes are welcome, maybe @EmanuelPerez or @patriziaazzi could test them?
We really need to replace the starter kits with edm4hep and remove all the old samples dictionaries etc, so feedback welcome!

### CODE REVIEW

**Comment by vvolkl** [comment]
Hi Clement,

great! Why not replace the old ones directly? They are archived in previous tags, and I'm pretty sure they won't work now anyway. Also, did you mean to add `FCCSnowmass.md` again?

**Comment by clementhelsens** [comment]
Thanks Valentin,

Will modify the pr accordingly !

Le 1 juin 2021 à 14:11, Valentin Volkl ***@***.******@***.***>> a écrit :


Hi Clement,

great! Why not replace the old ones directly? They are archived in previous tags, and I'm pretty sure they won't work now anyway. Also, did you mean to add FCCSnowmass.md again?

File: fast-sim-and-analysis/FccFastSimAnalysis.md [hunk 1]
```diff
@@ -23,71 +23,64 @@ git clone https://github.com/HEP-FCC/FCCAnalyses.git
...
 ```
 
-This tutorial runs with fcc-edm so we need to checkout the corresponding branch
-
-
-```bash
-git checkout fccedm
-```
-
 and follow the compilation instructions [here](https://github.com/HEP-FCC/FCCAnalyses/#getting-started) to get started with **ONLY** the installation.
```
Comment (vvolkl) [inline fast-sim-and-analysis/FccFastSimAnalysis.md:26]: in principle, FCCAnalyses is installed on cvmfs, so this should work without compiling anything, but maybe you also want people to know how it is done?

File: fast-sim-and-analysis/FccFastSimAnalysis.md [hunk 2]
```diff
@@ -23,71 +23,64 @@ git clone https://github.com/HEP-FCC/FCCAnalyses.git
...
 Once the code has been compiled, we can now run the pre-selection on previously produced samples:
 
 ```bash
-python FCCeeAnalyses/ZH_Zmumu/dataframe/analysis.py PATH_TO_FILES_PART_I/p8_ee_ZH_ecm240.root
-python FCCeeAnalyses/ZH_Zmumu/dataframe/analysis.py PATH_TO_FILES_PART_I/p8_ee_ZZ_ecm240.root
-python FCCeeAnalyses/ZH_Zmumu/dataframe/analysis.py PATH_TO_FILES_PART_I/p8_ee_WW_ecm240.root
+python examples/FCCee/higgs/mH-recoil/mumu/analysis.py PATH_TO_FILES_PART_I/p8_ee_ZH_ecm240.root
```
Comment (vvolkl) [inline fast-sim-and-analysis/FccFastSimAnalysis.md:30]: What is `PATH_TO_FILES_PART_I`?

File: fast-sim-and-analysis/FccFastSimAnalysis.md (hunk 1 above)
Comment (clementhelsens) [reply fast-sim-and-analysis/FccFastSimAnalysis.md:26]: this is true, for this tutorial where no code development is needed, we could remove the local installation. While a more advanced tutorial could be written to add a new algorithm. I can do it later. And BTW, in master a performance issue was fixed, I just made a new release, could we deploy it? I'd prefer if this tutorial depends on it
https://github.com/HEP-FCC/FCCAnalyses/releases/tag/v0.3.2

**Comment by vvolkl** [comment]
We can have these pages be checked automatically with the ci, but for this to work you need to assume each code cell starts from the same directory, i.e. any `cd` commands from previous commands are not in effect. Either the `cd` needs to be put at the beginning of each code cell, or env var can be used. I can try to do that.

File: fast-sim-and-analysis/FccFastSimAnalysis.md (hunk 2 above)
Comment (clementhelsens) [reply fast-sim-and-analysis/FccFastSimAnalysis.md:30]: this is the path to the file produced in part 1 (with k4SimDelphes) the two are somehow independent

**Comment by clementhelsens** [comment]
If we change this to use the cvmfs one, then we can run from the same directory

**Comment by vvolkl** [comment]
Replaced by #62

**Pull request closed without merging**